src/api/lifespan.py

```diff
@@ -6,28 +6,39 @@
 
 from src.config import api_settings
 from src.storage.sql import SQLAlchemyStorage
+import asyncio
+from src.api.use_cases.notifications import notification_use_case
 
 
 async def setup_repositories() -> SQLAlchemyStorage:
     from src.repositories.users.repository import user_repository
     from src.repositories.bookings.repository import booking_repository
     from src.repositories.auth.repository import auth_repository
     from src.repositories.innohassle_accounts import innohassle_accounts
+    from src.api.use_cases.notifications import notification_use_case
 
     storage = SQLAlchemyStorage.from_url(api_settings.db_url)
     user_repository.update_storage(storage)
     booking_repository.update_storage(storage)
     auth_repository.update_storage(storage)
 
     await innohassle_accounts.update_key_set()
+    asyncio.create_task(notification_use_case.notify_users_about_upcoming_bookings())
```

Comment:
Why do we need to call the method here?

Why not inside loop?

Reply:
forgot to delete this line after creation notifications loop